### PR TITLE
Typo, redundant string interpolation, namespaces and imports cleanup

### PR DESCRIPTION
**Pull request by DrSmugleaf**
Because antag is a word.

### CODE REVIEW

File: Content.Server/GameObjects/Components/Mobs/MindComponent.cs [hunk 1]
```diff
@@ -181,11 +177,11 @@ public void Examine(FormattedMessage message, bool inDetailsRange)
             if (!HasMind)
             {
                 message.AddMarkup(!dead
-                    ? $"[color=red]" +
+                    ? "[color=red]" +
                       Loc.GetString(
                           "{0:They} {0:are} totally catatonic. The stresses of life in deep-space must have been too much for {0:them}. Any recovery is unlikely.",
                           Owner) + "[/color]"
-                    : $"[color=purple]" + Loc.GetString("{0:Their} soul has departed.", Owner) + "[/color]");
+                    : "[color=purple]" + Loc.GetString("{0:Their} soul has departed.", Owner) + "[/color]");
```
Comment (exp111) [inline Content.Server/GameObjects/Components/Mobs/MindComponent.cs:184]: Same here

File: Content.Server/GameObjects/Components/Suspicion/SuspicionRoleComponent.cs [hunk 2]
```diff
@@ -188,8 +188,8 @@ void IExamine.Examine(FormattedMessage message, bool inDetailsRange)
             }
 
             var tooltip = IsTraitor()
-                ? Loc.GetString($"They were a [color=red]traitor[/color]!")
-                : Loc.GetString($"They were an [color=green]innocent[/color]!");
+                ? Loc.GetString("They were a [color=red]traitor[/color]!")
```
Comment (exp111) [inline Content.Server/GameObjects/Components/Suspicion/SuspicionRoleComponent.cs:191]: Just do `Loc.GetString("They were a [color={0}]{1}[/color]!",color,Loc.GetString(role))` or smth like that

File: Content.Server/GameTicking/GameRules/RuleDeathMatch.cs [hunk 3]
```diff
@@ -91,7 +91,7 @@ private void _checkForWinner()
                 _chatManager.DispatchServerAnnouncement($"{winner} wins the death match!");
             }
 
-            _chatManager.DispatchServerAnnouncement($"Restarting in 10 seconds.");
+            _chatManager.DispatchServerAnnouncement("Restarting in 10 seconds.");
```
Comment (exp111) [inline Content.Server/GameTicking/GameRules/RuleDeathMatch.cs:94]: We could Loc them... But who plays Deathmatch?

File: Content.Server/GameTicking/GameRules/RuleSuspicion.cs [hunk 4]
```diff
@@ -126,7 +125,7 @@ private void EndRound(Victory victory)
             }
 
             _gameTicker.EndRound(text);
-            _chatManager.DispatchServerAnnouncement($"Restarting in 10 seconds.");
+            _chatManager.DispatchServerAnnouncement("Restarting in 10 seconds.");
```
Comment (exp111) [inline Content.Server/GameTicking/GameRules/RuleSuspicion.cs:128]: But we should probably Loc those (and also maybe don't hardcode the restarttime?)

**Review by exp111 — CHANGES_REQUESTED** [review]

**Comment by DrSmugleaf** [comment]
Reviews addressed.

**Review by exp111 — APPROVED** [review]

**Pull request merged**